src/loaders/mongoose.js

```diff
@@ -34,4 +36,24 @@ const setupDbConnection = async () => {
     });
 };
 
+const createDefaultAdmin = async () => {
+    if (process.env.NODE_ENV === "test") {
+        return;
+    }
+
+    const acc = await Account.findOne({ email: config.admin_email });
+    if (acc) {
+        // User with admin email already exists
+        return;
+    }
```

Comment:
Should we instead check for an (any) admin user already existing?

Reply:
I don't think so, we want THAT admin to exist, we don't want to find via trial and error which admin existed before

Reply:
Ok, it was just checking for an admin with the given email vs checking for at least one admin being in the system. But this is fine for me as well. Resolving this thread then :slightly_smiling_face: